rocoto/py/GEFS_Crontab.py

```diff
@@ -20,7 +20,7 @@ def create_crontab(dicBase, OnlyForTest=False, cronint=5):
     ##########################################################
 
     # No point creating a crontab if rocotorun is not available.
-    import sys
+    import sys, os
```

Comment:
PEP-8 says one import per line
```suggestion
    import sys
    import os
```

Reply:
I also did changes for other scripts